bundler/lib/dependabot/bundler/file_parser.rb

```diff
@@ -32,9 +32,28 @@ def parse
         dependency_set.dependencies
       end
 
-      sig { returns(PackageManagerBase) }
+      sig { returns(Ecosystem) }
+      def ecosystem
+        @ecosystem ||= T.let(
+          Ecosystem.new(
+            name: ECOSYSTEM,
+            package_manager: package_manager,
+            language: language
+          ),
+          T.nilable(Ecosystem)
+        )
+      end
+
```

Comment:
We just updated existing package manager by wrapping it with ecosystem object and we added name for ecosystem and language information.